app/services/xccdf_report_parser.rb

```diff
@@ -98,10 +100,6 @@ def rule_results_rule_ids
     ).pluck(:id)
   end
 
-  def host_id
-    @host_id ||= Host.select(:id).find_by(name: report_host)
```

Comment:
This just seems a bit dangerous to me and unnecessary. Why query the DB if we already have the host in memory?